### DISCUSSION TITLE
How to give a small offset to markers

### DISCUSSION THREAD

**Discussion by sulhadin** [tradingview/lightweight-charts#1577] [Q&A]
Red arrows indicate two markers ATH and ATL points. The ATL point (at the left which is `18.564`) is cut. Is there any way to give an offset only to the marker but not the chart so that I can see the ATH point (price) whole without any cut?

a tiny note: The timescale props `fixLeftEdge` and `fixRightEdge` configurations  in my chart are set to `true` (I want it that way :) ).

```
///Version
"lightweight-charts": "^4.1.3",
```

![Screenshot 2024-05-03 at 17 35 09](https://github.com/tradingview/lightweight-charts/assets/1446064/738dfa22-d1fb-4920-b7f1-78145bbb60bc)

**Comment by sulhadin** [comment]
Here another example, any suggestion please?

![Chart data ATH ATL](https://github.com/tradingview/lightweight-charts/assets/1446064/bccb3f74-7c4a-4d9f-8819-61da5b1a4594)

**Comment by sulhadin** [comment]
anybody? :)

**Comment by sulhadin** [comment]
Hi, I wonder if anybody could answer this question.

**Comment by jedrok** [comment]
Try this solution found here. 
https://github.com/tradingview/lightweight-charts/issues/406
